### DISCUSSION TITLE
Font size extremely small and not proportionate

### DISCUSSION THREAD

**Discussion by notsheesh** [LekoArts/gatsby-themes#1047] [Questions] [answered]
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/42465144/201404371-f8cfbe91-4a94-44b8-b8da-dd73ffb6259d.png">

**Comment by torn4dom4n** [comment]
I have a same issue. It works when I build the site on Gatsby Cloud but doesn't work with GitHub page.
BTW, I use Gatsby v5.0.1 and the theme v5.

**Comment by LekoArts** [comment]
You'll need to give more details than a screenshot. https://minimal-blog.lekoarts.de/ is working fine

**Reply by LekoArts** [reply]
When e.g. looking at the heading in the footer, it gets this CSS:

```html
<style data-emotion="css 1c4kh09">
    .css-1c4kh09 {
        font-family: inherit;
        font-weight: 700;
        line-height: 1.25;
        margin: 0;
        margin-bottom: 0.25rem;
        font-size: 3px;
        margin-top: 0.5rem;
        color: var(--theme-ui-colors-heading);
        font-family: inherit;
        font-weight: 700;
        line-height: 1.25;
        margin: 0;
        margin-bottom: 0.25rem;
        font-size: 3px;
        margin-top: 0.5rem;
        color: var(--theme-ui-colors-heading);
    }

    @media screen and (min-width: 640px) {
        .css-1c4kh09 {
            font-size:4px;
        }
    }

    @media screen and (min-width: 640px) {
        .css-1c4kh09 {
            font-size:4px;
        }
    }
</style>
```

But the theme doesn't define that at all. I don't understand why your site deploy on GitHub actions does have that 🤷

**Comment by ghost** [comment]
I have same issue in gatsby-starter-portfolio-cara. The font's are way too small. @LekoArts

**Reply by shirgoldbird** [reply]
Also running into this issue with this starter. Fresh install of the Cara starter with Gatsby 5.0.0.

**Comment by LekoArts** [comment]
So for example in the emilia theme the fontSizes are defined with numbers: https://github.com/LekoArts/gatsby-themes/blob/7ff2d6f0c27acf31001a9244b066c614c32d5647/themes/gatsby-theme-emilia/src/gatsby-plugin-theme-ui/index.js#L84-L106

This is exactly how it's supposed to work: https://theme-ui.com/guides/responsive-typography

So `h1` has `5` at largest media query. The CSS has `5px` so for some reason in your case the `5` doesn't get replaced with the proper Theme UI value but with `5px`.

But no idea why it happens 🤷

**Reply by ghost** [reply]
In gatsby-starter-portfolio-cara, The CSS of your hosted site and manually developed sites using readme file are different. you can check yourself. @LekoArts

**Comment by LekoArts** [comment, marked as answer]
This should be fixed now. Delete your `node_modules` and lock file, re-install again.

**Reply by shirgoldbird** [reply]
Fixed for me! Thank you very very much.

**Reply by ghost** [reply]
It is fixed. thank you @LekoArts
